src/Psalm/Internal/Analyzer/Statements/Block/IfElseAnalyzer.php.orig

```diff
@@ -0,0 +1,447 @@
+<?php
```

Comment:
It looks like you accidentally committed this.

Reply:
Yep. And deleted locally already, just didn't push.